analyzers/tests/SonarAnalyzer.TestFramework/Verification/Verifier.cs

```diff
@@ -203,8 +203,14 @@ private IEnumerable<CompilationData> CompileRazor()
                     var csProjPath = PrepareRazorProject(projectRoot, langVersion);
                     var razorFiles = PrepareRazorFiles(projectRoot);
                     // To avoid reference loading issues, ensure that the project references are restored before compilation.
-                    Process.Start(Environment.GetEnvironmentVariable("MSBUILD_PATH"), $"restore {csProjPath}").WaitForExit();
-                    yield return new(workspace.OpenProjectAsync(csProjPath).Result.GetCompilationAsync().Result, razorFiles.ToArray());
+                    if (RestorePackages(csProjPath, projectRoot) == 0)
```

Comment:
Can you directly un-ignore this, so we prove it works and we won't need 2nd PR?
https://github.com/SonarSource/sonar-dotnet/pull/8653/files